.github/workflows/release.yml

```diff
@@ -0,0 +1,109 @@
+# This workflow will release the packages.
+
+name: 🚀 Release - 🤖 Android SDKs
+
+on:
+  workflow_dispatch:
+    inputs:
+      sdkType:
+        type: choice
+        description: Choose which sdk you want to release
+        default: 'android'
+        options:
+        - android
+        - android-core
+      version:
+        type: choice
+        description: Choose which version to bump before release
+        default: 'Patch'
+        options:
+        - Major
+        - Minor
+        - Patch
+
+env:
+    # GH_TOKEN: ${{ secrets.RELEASE_BOT_TOKEN }}
+    BRANCH: master
+    WORKING_DIR: ./android
+    # BOT_USERNAME: ${{ secrets.RELEASE_BOT_USER_NAME }}
+    # BOT_EMAIL: ${{ secrets.RELEASE_BOT_EMAIL }}
+
+jobs:
+  release:
+    name: 📦 Release
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
Same condition is duplicated?

Reply:
directly got this from the https://github.com/wso2/oxygen-ui/blob/f1cd341c8c6488e04bf9774b05e41d6ae45f9ea3/.github/workflows/release.yml#L33